src/QsCompiler/LlvmBindings/Instructions/InstructionBuilder.cs

```diff
@@ -106,7 +105,6 @@ public void PositionAtEnd(BasicBlock basicBlock)
         /// is not required to come from the same block the instruction builder is
         /// currently referencing.</note>
         /// </remarks>
-        [SuppressMessage("Microsoft.Design", "CA1011:ConsiderPassingBaseTypesAsParameters", Justification = "Specific type required by interop call")]
```

Comment:
Out of curiosity, what motivated removing these attributes?

Reply:
Visual studio, which told me they were unnecessary.